packages/host/app/lib/current-run.ts

```diff
@@ -278,8 +278,16 @@ export class CurrentRun {
 
       let { content, lastModified } = fileRef;
       if (url.href.endsWith('.json')) {
-        let { data: resource } = JSON.parse(content);
-        if (isCardResource(resource)) {
+        let resource;
+
+        try {
+          let { data } = JSON.parse(content);
+          resource = data;
+        } catch (e) {
+          log.warn(`unable to parse ${url.href} as card JSON`);
```

Comment:
at this point we can't even say that it is card json. probably a better warning is 
```
og.warn(`unable to parse ${url.href} as JSON`);
```